src/index.ts

```diff
@@ -177,22 +188,23 @@ function _decode(input: Buffer): Decoded {
   } else {
     // a list  over 55 bytes long
     llength = firstByte - 0xf6
-    length = safeParseInt(input.slice(1, llength).toString('hex'), 16)
+    length = safeParseInt(safeSlice(input, 1, llength).toString('hex'), 16)
+    if (length < 56) {
+      throw new Error('invalid rlp: encoded list too short')
```

Comment:
nit just to match the other error messages
```suggestion
      throw new Error('invalid RLP: encoded list too short')
```